cmd/root.go

```diff
@@ -64,7 +63,7 @@ A mutating webhook for Kubernetes, pointing the images to a new location.`,
 		//metricsRec := metrics.NewPrometheus(promReg)
 		log.Trace().Interface("config", cfg).Msg("config")
 
-		rClient, err := registry.NewECRClient(cfg.Target.AWS.Region, cfg.Target.AWS.EcrDomain())
+		rClient, err := registry.NewECRClient(cfg.Target.AWS.Region, cfg.Target.AWS.EcrDomain(), cfg.RepositoryCustomTags)
```

Comment:
How about moving this to a separate method, e.g. `setRepositoryCustomTags`?

Reply:
Hello,

Thanks for your comment. I'll do :-)

Rémi